### PR TITLE
models.yml: More clear enums

### PR DESCRIPTION
**Pull request by ostcar**

### CODE REVIEW

File: docs/models.yml [hunk 1]
```diff
@@ -212,18 +212,23 @@ user:
     fields:
       type: relation-list
       to: projector/current_element_ids
-  vote_delegated_$_to_id:
```
Comment (FinnStutzenstein) [inline docs/models.yml:215]: Please undo this change! The template form is required for requesting specific data from the autoupdate service. The format cannot deal with m2m tables spanning over meetings.

File: docs/models.yml [hunk 2]
```diff
@@ -715,9 +723,9 @@ meeting:
     type: relation
     to: group/default_group_for_meeting_id
     required: true
-  superadmin_group_id:
+  admin_group_id:
```
Comment (FinnStutzenstein) [inline docs/models.yml:726]: Why did you change this? it is already implemented in the backend and client...

**Review by FinnStutzenstein — CHANGES_REQUESTED** [review]

File: docs/models.yml (hunk 2 above)
Comment (FinnStutzenstein) [reply docs/models.yml:726]: But for me it is ok.

File: docs/models.yml (hunk 1 above)
Comment (normanjaeckel) [reply docs/models.yml:215]: @ostcar Please discuss this during code sprint. Only to be kept in mind: The problem has something to do with the structure of permission service (only yes or no) and restriction service (restrict relation id fields only if the user can not see any field of the related object). @FinnStutzenstein may give an example of an autoupdate request that requires this template form tomorrow.

Comment (ostcar) [reply docs/models.yml]: I removed the commit

**Review by FinnStutzenstein — APPROVED** [review]
